### PR TITLE
Add explanation of typescript test

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

na

## What does this implement/fix? Explain your changes.

Add explanation of typescript test

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,6 +86,12 @@ Note that `OPTUNA_DASHBOARD_DEBUG=1` makes the server will automatically restart
 
 ### Running tests, lint checks and formatters
 
+#### Running typescript unit tests
```
Comment (porink0424) [inline CONTRIBUTING.md:89]: I think this title is a little bit ambiguous (e.g., Are all typescript files to be tested??), but here you mean typescript unit tests on ts files in `tslib/`:

```suggestion
#### Running unit tests for `tslib/`
```

File: Makefile [hunk 2]
```diff
@@ -18,6 +18,11 @@ vscode/assets/bundle.js: $(RUSTLIB_OUT) $(STANDALONE_SRC) tslib
 $(DASHBOARD_TS_OUT): $(DASHBOARD_TS_SRC)
 	cd optuna_dashboard && npm install && npm run build:$(MODE)
 
+.PHONY: tstest
```
Comment (porink0424) [inline Makefile:21]: How about this:
```suggestion
.PHONY: tslib-test
```

**Review by porink0424 — COMMENTED** [review]
Thanks a lot for your PR 👍 I left some suggestions here.

**Comment by keisuke-umezawa** [comment]
@porink0424 Thank you! I fixed code as you said.

**Review by porink0424 — APPROVED** [review]
LGTM!

**Pull request merged**